### PR TITLE
Add tool dependency for bcftools 1.2

### PR DESCRIPTION
**Pull request by davebx**

### CODE REVIEW

**Comment by bgruening** [comment]
@davebx can you add a .shed.yml file please. Looks good! Thanks!

**Comment by erasche** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://gx.hx42.org//job/IUC-Github/53/
Test (probably) passed

**Comment by erasche** [comment]
@davebx example shed.yml for reference: https://gist.github.com/erasche/25a2405f259f9d2fb14f

**Comment by erasche** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://gx.hx42.org//job/IUC-Github/54/
Test (probably) passed

**Comment by martenson** [comment]
unfortunately there was an effort overlap and there is another implementation of this here: https://github.com/Takadonet/tools-iuc/commit/cd58375a543c67f987e1494742f3f156c1fb7ed8

**Comment by davebx** [comment]
@martenson @Takadonet it compiled on our build server without explicitly pulling in zlib.

**Comment by erasche** [comment]
@bgruening provisional suggestion: maybe we should move to requesting PRs to @natefoo's https://github.com/natefoo/docker-build for anything that's depends on packages added to depot (otherwise it's just a black box without build reproducibility).

**Comment by bgruening** [comment]
+1 for collecting this depot builds in @natefoo `docker-build` repository.

**Comment by davebx** [comment]
I agree wholeheartedly and will proceed with docker-build at my earliest convenience.

**Comment by Takadonet** [comment]
Sweet. Thanks again for adding the package

File: packages/package_bcftools_1_2/tool_dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                    <action type="move_directory_files">
+                        <source_directory>.</source_directory>
+                        <destination_directory>$INSTALL_DIR</destination_directory>
+                    </action>
+                </actions>
+                <actions>
+                    <action type="download_by_url">https://github.com/samtools/bcftools/releases/download/1.2/bcftools-1.2.tar.bz2</action>
+                    <action type="shell_command">sed -i.bak 's#/usr/local#$INSTALL_DIR#' Makefile</action>
+                    <action type="shell_command">make</action>
+                    <action type="shell_command">make install</action>
```
Comment (bgruening) [inline packages/package_bcftools_1_2/tool_dependencies.xml]: can be replaced by ` <action type="make_install"/>`

**Comment by bgruening** [comment]
@davebx this means zlib is needed?

**Comment by davebx** [comment]
@bgruening I just checked with docker-build, bcftools fails to compile if zlib is not installed.

**Comment by bgruening** [comment]
@Takadonet is it ok with you if I merge this one? Thanks @davebx and @Takadonet for working on this!

**Comment by Takadonet** [comment]
Go for it!

**Comment by davebx** [comment]
As always, I appreciate the feedback from @bgruening, @erasche, and @Takadonet. I'll be pushing this to the toolsheds post haste.

**Comment by erasche** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://gx.hx42.org//job/IUC-Github/80/
Test (probably) passed

**Comment by erasche** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://gx.hx42.org//job/IUC-Github/82/
Test (probably) passed

**Pull request merged**